src/cmds/qstat.c

```diff
@@ -294,6 +309,47 @@ states(char *string, char *q, char *r, char *h, char *w, char *t, char *e, int l
 	}
 }
 
+/**
+ * @brief
+ *	Prepend escape character with special characters such as line feed and form feed
+ *
+ * @param[in]   val - string which needs to be formatted
+ *
+ * @return string
+ * @retval NULL     - if memory allocation failed.
+ * @retval string   - formatted string.
+ *
+ */
+char *
+escape_feed_chars(char *val) {
+	int     i = 0;
+	int     len = MAXPATHLEN;
+	char    *temp = NULL;
+	char    *buf = (char *) malloc(MAXPATHLEN);
+	if(buf == NULL)
+		return NULL;
+	while(*val != '\0') {
+		if(*val == '\n') {
+			buf[i++] = '\\'; buf[i++] = 'n'; val++;
+		}
+		else if(*val == '\f') {
+			buf[i++] = '\\'; buf[i++] = 'f'; val++;
+		}
```

Comment:
variable_list containing a `\r` will not break encoding to dsv or one line output.